src/quo2/components/notifications/activity_logs.cljs

```diff
@@ -58,32 +59,30 @@
             :justify-content :center}
    [icon/icon icon {:color colors/white}]])
 
-(defn- activity-unread-dot
-  []
-  [rn/view {:margin-left      14
-            :margin-right     6
-            :background-color colors/primary-50
-            :width            8
-            :height           8
-            :border-radius    4}])
-
 (defn- activity-context
   [context replying?]
-  (let [first-line-offset (if replying? 4 -2)
+  (let [first-line-offset (if replying? 4 0)
         gap-between-lines 4]
-    (into [rn/view {:flex-direction :row
-                    :align-items    :center
-                    :flex-wrap      :wrap
-                    :margin-top     first-line-offset}]
-          (map-indexed (fn [index detail]
-                         ^{:key index}
-                         [rn/view {:margin-right 4
-                                   :margin-top   gap-between-lines}
-                          (if (string? detail)
-                            [text/text {:size :paragraph-2}
-                             detail]
-                            detail)])
-                       context))))
+    (into [rn/view {:style {:flex-direction  :row
+                            :flex            1
+                            :align-items     :center
+                            :justify-content :flex-start
+                            :flex-wrap       :wrap
+                            :margin-top      first-line-offset}}]
+          (mapcat (fn [detail]
+                    ^{:key (hash detail)}
+                    (if (string? detail)
+                      (map-indexed (fn [index s]
```

Comment:
what is `s`. Can it be more descriptive?

Reply:
`s` is a common name in Clojure that means it's a string, you can see this and other idiomatic names in https://github.com/bbatsov/clojure-style-guide#idiomatic-names.

Reply:
cool, the clojure guys are wrong imo 😆 but sounds good! 😁

Reply:
Yep, it's a questionable practice in other programming languages, but in Clojure it's very strong. Rich Hickey has many strong opinions about naming and we can see that in the Clojure core, his presentations, as well as how the community absorbed such patterns over many years.

Reply:
Sounds like I need to do some research on it :)